nengo_loihi/hardware/nxsdk_objects.py

```diff
@@ -318,7 +316,7 @@ def set_axons(self, board, nxsdk_board, spike_input):
                             axon.pop_type,
                             tchip.id,
                             tcore.id,
-                            taxon_id,
```

Comment:
My reasoning for writing it that way originally was to help make it clear what that parameter is/represents.

Reply:
It gets changed in #246 anyway which will be squashed in here so it gets changed back.